### PR TITLE
addition of KFCluster.CreatedAt

### PR DESCRIPTION
**Pull request by alessandroargentieri**

### CODE REVIEW

**Review by tahsin352 — APPROVED** [review]
LGTM

File: kfcluster.go [hunk 1]
```diff
@@ -5,18 +5,20 @@ import (
...
-	Namespace     string `json:"-"`
+	ID            string    `json:"id"`
+	Name          string    `json:"name"`
+	NetworkID     string    `json:"network_id"`
+	FirewallID    string    `json:"firewall_id"`
+	Size          string    `json:"size"`
+	KubeflowReady string    `json:"kubeflow_ready"`
+	DashboardURL  string    `json:"dashboard_url"`
+	CreatedAt     time.Time `json:"created_at"`
+	Namespace     string    `json:"-"`
```
Comment (RealHarshThakur) [inline kfcluster.go:21]: Namespace?

**Review by RealHarshThakur — CHANGES_REQUESTED** [review]
time.Time for CreatedAt? How is API sending this? How do we handle time in general?

File: kfcluster.go (hunk 1 above)
Comment (alessandroargentieri) [reply kfcluster.go:21]: I think is a refuse. I'll remove it. It caught my attention too

**Comment by alessandroargentieri** [comment]
> 

API sends `CreatedAt` field as `time.Time` for every resource (e.g. instances, clusters, DNSRecords, Accounts,...)

**Review by RealHarshThakur — APPROVED** [review]

**Pull request merged**